src/day_7.rs

```diff
@@ -30,10 +30,10 @@ fn main() {
                     .iter()
                     .enumerate()
                     .map(|jj| {
-                        vv[jj.0 as usize] = (*hm
+                        [vv[jj.0 as usize] = (*hm
                             .entry((ii.max(*jj.1), ii.min(*jj.1)))
                             .or_insert(ii.max(*jj.1) - ii.min(*jj.1)))
-                            as isize;
+                            as isize][0]
```

Comment:
Here's something I haven't showed before, normally I use an iterator for this kind of stuff but this will won't even cause clippy to complain.

Reply:
nice technique :) I will keep this suggestion in my mind while working on next part :100: